AngleSharp.Scripting.CSharp/AngleSharp.Scripting.CSharp.csproj

```diff
@@ -43,8 +43,9 @@
     <Compile Include="WindowService.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="AngleSharp">
+    <Reference Include="AngleSharp, Version=0.8.7.42712, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Sorry this could be specific to my development environment....  Didn't mean to include it in the pull request.

For some reason the nuget packages wouldn't restore when I did the original checkout (VS 2015 final).  I ran "Update-Package -Reinstall" which fixed this and I presume made the above modifications (and the below "private" one)